packages/components-js/package.json

```diff
@@ -46,6 +49,12 @@
     "test:e2e:playwright": "playwright test --config=tests/e2e/playwright/config/playwright.config.ts",
     "test:vrt:puppeteer": "TYPE=VRT PORT=8575 jest --colors -c tests/vrt/puppeteer/config/jest.config.ts",
     "test:vrt:playwright": "playwright test --config=tests/vrt/playwright/config/playwright.config.ts",
+    "test:vrt:playwright:ui": "yarn test:vrt:playwright --ui",
+    "test:vrt:playwright:chrome:common": "yarn test:vrt:playwright common --project=chrome",
+    "test:vrt:playwright:chrome:states": "yarn test:vrt:playwright states --project=chrome",
+    "test:vrt:playwright:chrome:custom": "yarn test:vrt:playwright custom --project=chrome",
+    "test:vrt:playwright:safari": "yarn test:vrt:playwright --project=safari",
+    "test:vrt:playwright:chrome:custom-states": "yarn test:vrt:playwright:chrome:custom && yarn test:vrt:playwright:chrome:states",
```

Comment:
Maybe rename to `customAndStates` otherwise one might be wondering which custom states.